BundleClient/app/src/main/java/net/discdd/bundleclient/BundleClientWifiDirectService.java

```diff
@@ -221,17 +224,31 @@ private BundleExchangeCounts exchangeWith(WifiP2pDevice device) {
         }
         try {
             var newGroup = connectTo(device).get(10, TimeUnit.SECONDS);
-            return bundleTransmission.doExchangeWithTransport(device.deviceAddress, device.deviceName,
+
+            BundleExchangeCounts currentBundle = bundleTransmission.doExchangeWithTransport(device.deviceAddress, device.deviceName,
                                                               wifiDirectManager.getGroupOwnerAddress().getHostAddress(),
                                                               7777);
+
+
+            final String text1 = currentBundle.uploadStatus() == Statuses.COMPLETE ? "Bundle has Been Completely Uploaded":"Bundle Upload has failed";
```

Comment:
can you change these to translatable strings so that translation works? (the IDE will do it for you.)